openclsim/appendix/appendix.py

```diff
@@ -0,0 +1,63 @@
+# ----------------------------------------------------------------------------!
+import pandas as pd
+
+
+# ----------------------------------------------------------------------------!
+def remove_item(d: dict, k: str or list):
```

Comment:
Maybee something to think about is recursion. If you have a dict in a dict in a dict can we also remove all the keys from nested dicts?

Not nessescary but fun to think about